File: benchmark/results/java-multilang/run-004/b-baseline/output/com/metaengine/demo/notification/services/DeliveryDispatcher.java
package com.metaengine.demo.notification.services;

import java.util.List;

import com.metaengine.demo.notification.aggregates.Notification;

/** DeliveryDispatcher domain service. */
public class DeliveryDispatcher {

    /** Dispatch a Notification for delivery. */
    public Notification create(Notification input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a Notification by id (may return null). */
    public Notification findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List notifications up to limit. */
    public List<Notification> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete a Notification by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-invocation-script/run-001/a-mcp/output/notification/services/delivery_dispatcher.py
from typing import List, Optional
from notification.aggregates.notification import Notification

"""DeliveryDispatcher service."""
class DeliveryDispatcher:
    def create(self, input: Notification) -> Notification:
        raise NotImplementedError('not implemented')
    def findById(self, id: str) -> Optional[Notification]:
        raise NotImplementedError('not implemented')
    def list(self, limit: int) -> List[Notification]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

